plugins/module_utils/cmd_runner_fmt.py

```diff
@@ -78,8 +78,10 @@ def func(value):
     return _ArgFormat(func, ignore_none=ignore_none)
 
 
-def as_fixed(args):
-    return _ArgFormat(lambda value: _ensure_list(args), ignore_none=False, ignore_missing_value=True)
+def as_fixed(*args):
+    if len(args) == 1 and is_sequence(args[0]):
+        args = args[0]
```

Comment:
I think the `if` is dangerous, as for a single argument the function already accepted both lists and not lists (and converted not lists to lists in `_ensure_list`).

Reply:
Err, the `if` is exactly what guarantees backwards compatibility

Reply:
You're right... I totally misread this...